src/character.cpp

```diff
@@ -11213,13 +11204,11 @@ int Character::temp_corrected_by_climate_control( int temperature ) const
             temperature = BODYTEMP_VERY_HOT;
         } else if( temperature > BODYTEMP_VERY_HOT ) {
             temperature = BODYTEMP_HOT;
-        } else if( temperature > BODYTEMP_HOT ) {
-            temperature = BODYTEMP_NORM;
         } else if( temperature < BODYTEMP_FREEZING ) {
             temperature = BODYTEMP_VERY_COLD;
         } else if( temperature < BODYTEMP_VERY_COLD ) {
             temperature = BODYTEMP_COLD;
-        } else if( temperature < BODYTEMP_COLD ) {
+        } else if( temperature > BODYTEMP_HOT || temperature < BODYTEMP_COLD ) {
```

Comment:
```suggestion
        } else if( temperature >= BODYTEMP_HOT || temperature <= BODYTEMP_COLD ) {
```

Reply:
Any particular reason you suggested this change specifically?  I think what I had written was preserving the old behaviour.

However, on further reflection, I think we can actually go even simpler and just change this case to an `else` with no condition.  That is a small change to behaviour, but it seems a reasonable one for what this function is supposed to do.